<?xml version="1.0"?>
<!--
run.launch: this file does no more than calling other launch file
Author: Levi O. Vasconcelos
Date: 2020/09/26
-->
<launch>
  <include file="$(find light_tracker)/launch/core.launch">
      <arg name="detector_input_topic" value="/face_mask/input/compressed"/>
      <arg name="tracker_input_topic" value="/tracker/input/compressed"/>
      <arg name="image_out_topic" value="/face_tracker/output/compressed"/>
      <arg name="add_entity_topic" value="/tracker/input/add"/>
      <arg name="tracker_out_topic" value="/tracker/output/entities"/>
      <arg name="detector_out_topic" value="/face_mask/output/faces"/>
      <arg name="camera_image_topic" value="/head_front_camera/image_raw/compressed"/>
   </include>
</launch>
